easybuild/easyconfigs/l/LHAPDF/LHAPDF-6.5.3-gompi-2022a.eb

```diff
@@ -0,0 +1,44 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Alexander Puck Neuwirth <alexander@neuwirth-informatik.de>
+
+easyblock = 'ConfigureMake'
+
+name = 'LHAPDF'
+version = '6.5.3'
+
+homepage = 'http://lhapdf.hepforge.org/'
+description = """
+Les Houches Parton Density Function unified library
+
+First you need to adjust by `export LHAPDF_DATA_PATH=/local/.../path/` to your.
+Then you can run `lhapdf update` followed by `lhapdf install your_pdf`.
```

Comment:
The need for `LHAPDF_DATA_PATH` is in the command like `lhapdf install cteq66` which fetches the cteq66 data. 
By default `/usr/share/LHAPDF` is used, which won't work for users in easybuild.

If I understand the AlphaFold configuration right the admin of the cluster then would have to provide a directory where everyone has write access or the data is already available.
I think the best option is to both print the message and the option for the admin to provide a directory with preinstalled data.

Reply:
Okay, thanks for clearing that up. With AlphaFold most users are using the same (huge) dataset, which is typically provided by admins in some standard location, and you can use only one directory. That's slightly different here, if I understand correctly, so in this case I'd put the `LHAPDF_DATA_PATH` in `modextrapaths`; that does support multiple paths separated by a colon (and it will append the one you put in the module file to existing ones), while `modextravars` does not and will just override whatever is already set.